uber/templates/groups/form.html

```diff
@@ -196,13 +196,14 @@
             <textarea rows="3" cols="30" name="special_needs">{{ group.special_needs }}</textarea>
         </div>
     </div>
```

Comment:
This doesn't affect MAGFest directly, so I don't care that much, but I'd be tempted to insert a note on the Attendee address when the dealer form is what's being loaded indicating that the address they enter MUST be their business address for tax purposes.  I can imagine plenty of situations where someone would enter their personal address instead of their business address (like if they expect the badge might get mailed to them).

Alternatively, it would probably be even better to show this field but make it optional when ``c.COLLECT_FULL_ADDRESS`` is turned on, with a note that you ONLY need to fill this out if your business' address is different from your personal address.  Kind of like how all the good online stores take your shipping address and then only make you fill out the billing address if it's different from the shipping address.

There are probably a LOT of businesses which are Delaware LLCs, which might mean a Delaware address for tax purposes.  But if an event was mailing out badges, someone would really NEED to put their own personal address.  So letting them enter both if they need to would be better.

But like I said, that doesn't affect MAGFest, so I'm okay with it being broken in that case, and an event which actually needed this could add it in their event-specific plugin if they wanted.